src/main/java/org/kjy5/Main.java

```diff
@@ -120,8 +131,10 @@ public static void main(String[] args) {
             nodeToSourceFileMapping,
             contentTupleToSourceFileMapping,
             ChildListVirtualNodesMapping);
-    System.out.println("State\tPCSs\tContentTuples");
+    System.out.println("State\t# PCSs\t# ContentTuples");
     System.out.println(
+        // Why are there two "\t" in a row here, but there was only one in the header line that was
+        // printed just above?
```

Comment:
This was just to make sure the text lined up (one `\t` was not enough). Do you know a better way to print tabular information in the console for debugging?

Reply:
Tabs have different widths in different environments.  If I want something that is machine-readable, the tab widths don't matter (and I can have my editor adjust tab widths as needed).  If it's for human readability, spaces are generally better.